src/App.js

```diff
@@ -1,13 +1,41 @@
-import React from 'react';
-// import { Counter } from './features/counter/Counter';
-// import './App.css';
```

Comment:
- [ ] Let's not use commented code and write clean code.

Reply:
Commented code removed.